sdkit/utils/memory_utils.py

```diff
@@ -106,64 +156,87 @@ def print_largest_tensors_in_memory(device, num=10):
     print(f'== {num} largest tensors on {device} ==')
     print(_fmt_tensors_summary(entries))
     print('---')
-    print(f'Total memory occupied on {device} by {n} tensors: {total_mem:.1f} MiB')
+    print(f'Total memory occupied on {device} by'
```

Comment:
Why the break of lines? We don't need to enforce 80-char line width, most monitors and IDEs easily show wider lines, and we don't need to make the codebase look artificially verbose.